src/Microsoft.ML.Legacy/Runtime/EntryPoints/JsonUtils/ExecuteGraphCommand.cs

```diff
@@ -188,28 +188,28 @@ public void GetOutputToPath(GraphRunner runner, string varName, string path, Tlc
 
         }
 
-        private void SaveArrayToFile<T>(List<T> array, TlcModule.DataKind kind, string partialPath, string extension)
-           where T : class
+        private void SaveArrayToFile(PredictorModel[] array, string partialPath, string extension)
```

Comment:
Wow, that's an interesting method. I wonder how it came to being?..

Reply:
No idea. This was one of the few non-mechanical changes in this PR. Obviously I simply could not let this stand when I stumbled across it. 😄